### DISCUSSION TITLE
Persistent SSH tunnel

### DISCUSSION THREAD

**Discussion by VeRychard** [Eugeny/tabby#2757] [General]
I couldn't find any doc on the SSH tunnel, only that it's implemented, so I already had a hard time to find it.
I have found it here: after I connect, there is a 3 dotted menu in the top right corner, which on hover changes into "Ports"  (could be just ports by default for straightforwardness)

My question is, how can I achieve so the tunnels persist after disconnecting from the server, and I don't have to manually type it again?

Thanks in advance.

**Comment by Eugeny** [comment]
An SSH "tunnel" can only exist within an active SSH connection. If you're looking to use a tunnel to connect to other hosts through it, you can choose it as a jump host for that connection and Terminus will manage the tunnel connection automatically.

**Reply by VeRychard** [reply]
Thanks, although I've found out there is already a [feature request ](https://github.com/Eugeny/terminus/issues/2395) about this, so I'm gonna have to wait for it :)
